conformance_tests/sysman/test_sysman_device/src/test_sysman_device_helper.cpp

```diff
@@ -96,8 +96,16 @@ int main(int argc, char **argv) {
     exit(1);
   }
   LOG_INFO << "Child Process : Sysman initialized";
-
-  auto sysman_devices = lzt::get_zes_devices();
+  auto zes_drivers = lzt::get_all_zes_driver_handles();
+  zes_driver_handle_t zes_driver_used;
+  for (const auto &zes_driver : zes_drivers) {
+    auto device_count = lzt::get_zes_device_count(zes_driver);
+    if (device_count > 0) {
+      zes_driver_used = zes_driver;
+      break;
+    }
+  }
+  auto sysman_devices = lzt::get_zes_devices(lzt::get_zes_device_count(zes_driver_used), zes_driver_used);
```

Comment:
can we simplify this to  call `get_zes_devices(get_device_count(zes_drivers[0]), zes_drivers[0])` ?
and add a comment saying we are using first driver found for  purpose of this test?